### PR TITLE
Directions 2.8

### PR DESCRIPTION
**Pull request by chezzdev**

### CODE REVIEW

File: Package.resolved [hunk 1]
```diff
@@ -42,17 +42,17 @@
...
       },
       {
         "package": "MapboxMobileEvents",
         "repositoryURL": "https://github.com/mapbox/mapbox-events-ios.git",
         "state": {
           "branch": null,
-          "revision": "91ae071c0232c144251dfe0d0724525deaf293d4",
-          "version": "1.0.10"
+          "revision": "5e130fba637f5bbb0bb8bca0dec8d38a06ce07bf",
+          "version": "1.0.8"
```
Comment (kried) [inline Package.resolved:55]: Looks like unexpected downgrade

[the remark this answers is not in the document]
Comment (chezzdev) [reply Package.resolved:55]: I thought so too, but there was a downgrade on the Maps side https://github.com/mapbox/mapbox-maps-ios/pull/1579/files.

File: Package.resolved [hunk 2]
```diff
@@ -132,8 +132,8 @@
         "repositoryURL": "https://github.com/apple/swift-argument-parser",
         "state": {
           "branch": null,
-          "revision": "fddd1c00396eed152c45a46bea9f47b98e59301d",
-          "version": "1.2.0"
+          "revision": "9f39744e025c7d377987f30b03770805dcb0bcd1",
+          "version": "1.1.4"
```
[the remark this answers is not in the document]
Comment (chezzdev) [reply Package.resolved:136]: What do you meed by not needed? This is how it's resolved 🤔

File: Cartfile.resolved [hunk 3]
```diff
@@ -2,7 +2,7 @@ binary "https://api.mapbox.com/downloads/v2/carthage/mapbox-common/MapboxCommon.
 binary "https://api.mapbox.com/downloads/v2/carthage/mobile-navigation-native/MapboxNavigationNative.xcframework.json" "119.0.2"
 github "Quick/Nimble" "v9.2.1"
 github "Quick/Quick" "v3.1.2"
-github "mapbox/mapbox-directions-swift" "v2.8.0-rc.2"
+github "mapbox/mapbox-directions-swift" "v2.8.0"
 github "mapbox/mapbox-events-ios" "v1.0.10"
```
Comment (kried) [inline Cartfile.resolved:6]: Should there also be 1.0.8? Carthage project doesn't require Maps so `mapbox-events-ios` is not downgraded herw, but I think we can get something unexpected if we build with 1.0.10 in one project and with 1.0.8 in another.

Comment (chezzdev) [reply Cartfile.resolved:6]: Ok, turns out I needed to call "Update to Latest Package Versions" in Xcode to align all the dependencies. Now SPM prefers 1.0.10 too.

**Comment by chezzdev** [comment]
> Changelog probably needs to be updated as well?

You're right, thanks!

File: Package.resolved (hunk 2 above)
[the remark this answers is not in the document]
Comment (chezzdev) [reply Package.resolved]: Resolved by updating to the latest version through Xcode.

**Review by kried — APPROVED** [review]

**Pull request merged**